phac-nml/irida-next
# Move component js to javascript/controllers folder to eliminate console errors

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Move view component javascript files into the `javascript/controllers` directory to stop getting errors that controllers could not be found in the browser console.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

N/A

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Ensure no error messages logged to the console and that controllers still function - tests should cover this.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
